reflex/event.py

```diff
@@ -1020,13 +1055,15 @@ def parse_args_spec(arg_spec: ArgsSpec):
     spec = inspect.getfullargspec(arg_spec)
     annotations = get_type_hints(arg_spec)
 
-    return arg_spec(
-        *[
-            Var(f"_{l_arg}").to(
-                unwrap_var_annotation(resolve_annotation(annotations, l_arg))
-            )
-            for l_arg in spec.args
-        ]
+    return list(
```

Comment:
why does this return a `list` now? this is the only change i don't really understand.

Reply:
because argspec is now typed as "sequence"